cherrypy/lib/static.py

```diff
@@ -75,6 +75,13 @@ def serve_file(path, content_type=None, disposition=None, name=None,
         i = path.rfind('.')
         if i != -1:
             ext = path[i:].lower()
+            if ext in mimetypes.encodings_map.keys():
```

Comment:
Sure, I could add a comment with something like:

> Set the content-type based on the real filename extension
> not the compression type. Also set the content-encoding
> for compressed files.

Would that suffice, or should it be reworded?